packages/nocodb/src/db/BaseModelSqlv2.ts

```
@@ -1845,6 +1845,22 @@
+              column,
+              columns,
+            );
+            columnQuery = this.dbDriver.raw('(??)::jsonb', [defaultColumnName]);
+            if (!isSubGroup) {
+              selectors.push(
+                this.dbDriver.raw(`?? as ??`, [columnQuery, alias]),
+              );
+            }
+            break;
+          }
+        }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)